content/en/docs/Getting started/setting-up-the-database.md

```diff
@@ -10,6 +10,18 @@ description: >
 This page describes how to install and set up a
 [PostgreSQL](https://www.postgresql.org/) database for use with OpenCue.
 
+OpenCue supports a single database per deployment, which serves as the single source of truth
+within your deployment.
+
+## System requirements
+
+*   [Cuebot](/docs/getting-started/deploying-cuebot) servers are the only
```

Comment:
As it's just a single item, I recommend removing the list formatting here.

Reply:
Done.